index.js

```diff
@@ -91,10 +92,11 @@ const createMeasurement = async (req, res, client, getCurrentRound) => {
         indexer_result,
         miner_id,
         provider_id,
-        completed_at_round
+        completed_at_round,
+        station_id
```

Comment:
Let's avoid code churn. (It's a pity that standard.js does not allow trailing commas here 😢 )

```diff
   provider_id,
+  station_id,
   completed_at_round
```

Personally, I would put this new item right after `participant_address`, but that's a nitpick - feel free to ignore it.